qredshift@quintao/files/qredshift@quintao/metadata.json

```diff
@@ -1,9 +1,9 @@
 {
   "uuid": "qredshift@quintao",
   "name": "QRedshift",
-  "version": "1.7.3",
+  "version": "1.7.4",
   "description": "It makes the color of your computer's display adapt to the time of day, warm at night and like sunlight during the day.",
   "multiversion": true,
   "max-instances": 1,
-  "cinnamon-version": ["3.8", "4.0", "4.2", "4.3", "4.4", "4.5", "4.6", "4.8", "5.0", "5.2", "5.4"]
+  "cinnamon-version": ["3.8", "4.0", "4.2", "4.3", "4.4", "4.5", "4.6", "4.8", "5.0", "5.2", "5.4", "5.5", "5.6"]
```

Comment:
This doesn't need 5.5 (since it doesn't exist), but you could add 5.8 (in addition to 5.6) as it will be out next month or so and should still be compatible.

Reply:
Since about cinnamon 5.0 I think, these cinnamon version numbers no longer need to be updated as all xlets are assumed to be forward compatible.

Reply:
So i will leave it as it is, thanks guys.